src/components/SymptomsCarousel/SymptomsCarousel.scss

```diff
@@ -0,0 +1,30 @@
+.Main {
+  width: 100%;
+  &__title {
+    font-size: 16px;
+    font-weight: bold;
+    color: #929292;
+    padding-left: 10px;
+  }
+}
+.Carousel {
+  overflow: scroll;
+  text-align: center;
+  &__title {
+    font-size: 20px;
+    font-weight: 900;
+    text-align: center;
+    color: #22577A;
+  }
+  &__container {
+    white-space: nowrap;
+  }
+}
```

Comment:
**Here a suggest some changes:**
- Remove title margins.
- Remove scroll bar visualization in chrome browsers when viewport doesn't cut the items.
- Add padding to `.Carousel` to prevent shadow cutting.

```suggestion
.Carousel {
  padding: 1rem 0;
  overflow: scroll;
  text-align: center;
  &::-webkit-scrollbar {
    display: none;
  }
  &__title {
    font-size: 20px;
    font-weight: 900;
    text-align: center;
    color: #22577a;
    margin: 0;
  }
  &__container {
    white-space: nowrap;
  }
}
```